tests/projects/test_entry_point.py

```diff
@@ -0,0 +1,92 @@
+import os
```

Comment:
This file's contents largely copied from the old `tests/projects/test_projects.py` (just changed to use the `TempDir` helper function)